### PR TITLE
V1.1.0 release notes

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) updates the release notes for version 1.1.0.

## Affected pages

- ...

## Links to Magento Open Source code

- ...

whatsnew
Updated the [release notes](https://developer.adobe.com/commerce/events/get-started/release-notes/) for V1.1.0.

### CODE REVIEW

File: src/pages/get-started/release-notes.md [hunk 1]
```diff
@@ -7,6 +7,20 @@ details: This page lists new features and known issues for each release of Adobe
...
+
+April 20, 2023
+
+### Enchancements
+
+Added support for delivering events using message queues. Previously, all events were delivered by cron, which could delay delivery by up to a minute. Use the `bin/magento events:subscribe --priority` command to register the event as requiring expedited delivery. See [Configure Adobe Commerce](./configure-commerce.md/#check-cron-and-message-queue-configuration) for information about configuring the message queues.
+
+### Bug fixes
+
+* Made multiple minor bug fixes.
```
Comment (oshmyheliuk) [inline src/pages/get-started/release-notes.md:22]: I would add some descriptions to important fixes:
```
- fixed conversion of the nested array of objects, previously it was missed in the final payload
- fixed displayed event information for multiple events that are based on DataObjects
- fixed performance degradation during event saving
- several other bug fixes
```

File: src/pages/get-started/release-notes.md [hunk 2]
```diff
@@ -7,6 +7,20 @@ details: This page lists new features and known issues for each release of Adobe
...
 
+## Version 1.1.0
+
+### Release date
+
+April 20, 2023
+
+### Enchancements
+
+Added support for delivering events using message queues. Previously, all events were delivered by cron, which could delay delivery by up to a minute. Use the `bin/magento events:subscribe --priority` command to register the event as requiring expedited delivery. See [Configure Adobe Commerce](./configure-commerce.md/#check-cron-and-message-queue-configuration) for information about configuring the message queues.
```
Comment (oshmyheliuk) [inline src/pages/get-started/release-notes.md:18]: Also added a new feature, for field filtering for an array of nested objects.  

```
items[].sku
items[].qty
```
Previously it was possible only to add `items` field and that would return all content of `items` array without filtration.


https://github.com/AdobeDocs/commerce-events/pull/121/files#diff-4bbca1891cbffbe0985d7092411f8a79ff44bf43be9e65a769b0bd85ed93e8f5L215

**Review by oshmyheliuk — APPROVED** [review]

File: src/pages/get-started/release-notes.md [hunk 3]
```diff
@@ -7,6 +7,34 @@ details: This page lists new features and known issues for each release of Adobe
...
+   ```text
+   items[].sku
+   items[].qty
+   ```
+
+Previously, you could not specify individual fields within an array. All fields within the array would be returned. [Commerce module development](module-development.md#array-of-nested-objects) demonstrates this feature in full context.
+
+### Bug fixes
+
+*  Fixed conversion of an nested array of objects. Previously they were omitted in the final payload.
```
Comment (jeff-matthews) [inline src/pages/get-started/release-notes.md:33]:
```suggestion
*  Fixed conversion of a nested array of objects. Previously they were omitted in the final payload.
```

**Review by jeff-matthews — CHANGES_REQUESTED** [review]
One minor change request

**Review by jhadobe — APPROVED** [review]

**Pull request merged**